pkgs/development/compilers/c3c/default.nix

```diff
@@ -50,11 +51,19 @@ llvmPackages.stdenv.mkDerivation rec {
     runHook postCheck
   '';
 
+  passthru.tests = {
+    version = testers.testVersion {
+      package = finalAttrs.finalPackage;
+      command = "${lib.getExe finalAttrs.finalPackage} --version";
```

Comment:
`command` can be omitted here
https://github.com/NixOS/nixpkgs/blob/e0c43a96d20d8604b84f07d016aeb90e2ff0e437/pkgs/build-support/testers/default.nix#L64

Reply:
Removed in latest push.